pygfx/resources/_texture.py

```diff
@@ -248,6 +248,7 @@ def format_from_memoryview(mem, size):
         "h": "i2",
         "H": "u2",
         "i": "i4",
+        "I": "i4",
         "U": "u4",
```

Comment:
I think you're right. Memoryview uses the struct format characters, and there is not "U" there: https://docs.python.org/3/library/struct.html#format-characters

Probably a typo from my end.

But should it not be like this then?
```suggestion
        "I": "u4",
```

Reply:
ah, seems right, didn't see the u2 and u1. Is there a table of the format mappings specified by pygfx? 

related, what texture formats are actually supported, int8 and int16 seem to work but [`WGSL`](https://www.w3.org/TR/WGSL/#texture-builtin-functions) seems to only specify int32, uint32, and float32, but wgpu supports more? https://docs.rs/wgpu/latest/wgpu/enum.TextureFormat.html

Does uint8 for example get converted to uint32 before writing to the GPU?